printer_mods/README.md

```diff
@@ -12,6 +12,7 @@ Discord to have your mod moved to this folder.
 ## Table structure
 
 Update the following table with the information about your mod:
+
```

Comment:
Please limit changes in this README to only your own entry. there are several unrelated changes here.

Reply:
Sorry, my editor is set up to automatically fix violations of the markdown style guide. I've fixed this.